src/main/java/core/basesyntax/entity/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.entity;
+
+import core.basesyntax.Colors;
+import core.basesyntax.Figure;
+import java.util.Random;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle() {
+        this.radius = new Random().nextInt(MAX_VALUE);
```

Comment:
There is no need to create random data within the constructors. It should be done in FigureSupplier and then passed to constructors of required figures